.travis.yml

```diff
@@ -32,6 +32,10 @@ before_install:
         conda install --yes scipy matplotlib nose coverage setuptools;
         travis_retry pip install -q coveralls;
       fi;
+    # install AVbin
+    - wget https://github.com/downloads/AVbin/AVbin/install-avbin-linux-x86-64-v10 -O avbin.sh
```

Comment:
Thanks, I'll look into that. How did you figure out it was ssh? All I saw
in the logs was "error code 4" which is supposedly a generic network error.
How could I have dug deeper?
On Mar 2, 2016 7:28 AM, "Eric Larson" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/LABSN/expyfun/pull/270#discussion_r54738099>:
>
> > @@ -32,6 +32,10 @@ before_install:
> >          conda install --yes scipy matplotlib nose coverage setuptools;
> >          travis_retry pip install -q coveralls;
> >        fi;
> > +    # install AVbin
> > +    - wget https://github.com/downloads/AVbin/AVbin/install-avbin-linux-x86-64-v10 -O avbin.sh
>
> The error is Unable to establish SSL connection, which suggests that it's
> an SSL protocol failure. Either a bug in Ubuntu/wget or GitHub's certs. Can
> you use http?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LABSN/expyfun/pull/270/files#r54738099>.
>